api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -369,24 +369,15 @@ private void setDbWorkspaceFields(
   public ResponseEntity<EmptyResponse> deleteWorkspace(
       String workspaceNamespace, String workspaceId) {
     recordOperationTime(
-        () -> deleteWorkspaceImpl(workspaceNamespace, workspaceId), "deleteWorkspace");
+        () -> {
```

Comment:
BTW, most of these are coming out soon in favor of the interceptor approach. I'm keeping them in place for a week or so since they're one release ahead and I need to get automation working on the logs-based stuff.